mevislab.github.io/content/tutorials/basicmechanisms.md

```diff
@@ -197,22 +193,26 @@ More information on module groups can be found {{< docuLinks "/Resources/Documen
-
-We now like to condense the modules forming the contour filter into one single macro module. To do that, right-click on the group-title and select *Convert To Local Macro*. Now, you need to select a name for your macro module and finish the creation of the module. With this you create a local macro module. Local macro modules can only be used in your current network.
+You have probably already noticed how the modules differ in color. Each color represents another type of module:
+  
+  * The blue modules are called ML-Modules, they process voxel-images. 
+  * Green modules are OpenInventor-modules, they enable visual 3D scene graphs. 
+  * The brown modules are called Macro-modules. Macro-modules encapsulate a whole network in a single module. 
+  
+To condense our filter into one single module, we will now be creating a Macro-module out of it. To do that, right-click on the group-title and select *Convert To Local Macro*. Name your new Macro-module and finish. You just created a local Macro-module. Local Macro-modules can only be used in the network they were created in.
 
 ![Convert to Local Macro](/images/tutorials/basicmechanics/BM_21.png "Convert to Local Macro")
 ![Your first Local Macro](/images/tutorials/basicmechanics/BM_22.png "Your first Local Macro")
 
-Right-click the macro module and select *Show Internal Network* to inspect and change the internal network. You can change the properties of the new macro module by changing the properties in the internal network. As before, you can for example right-click the module `Convolution` and change the Kernel. These changes will be preserved.
+Right-click the Macro-module and select *Show Internal Network* to inspect and change the internal network. You can change the properties of the new Macro-module by changing the properties in the internal network. You can, for example, right-click the module `Convolution` and change the kernel. These changes will be preserved.
 
-![Internal Network of your Local Macro](/images/tutorials/basicmechanics/BM_23.png "Internal Network of your Local Macro")
+![Internal Network of your Local Macro](/images/tutorials/basicmechanics/BM_23.png "Internal Network of your local Macro")
 
 {{< youtube "VmK6qx-vKWk">}}
 
 {{<alert class="info" caption="Extra Infos">}}
 Module handling is explained {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch03s09.html" "here" >}}
-More information about Marco Modules can be found {{< docuLinks "/Resources/Documentation/Publish/SDK/GettingStarted/ch09.html" "here" >}}
+More information on Macro-modules can be found {{< docuLinks "/Resources/Documentation/Publish/SDK/GettingStarted/ch09.html" "here" >}}
```

Comment:
macro modules, also below

Reply:
Sure, will take some time to let VS Code substitute every "Macro Module" or "Macro module" with "macro module".

Reply:
Tried my best to replace everything that was upper case with lower case, we'll see if I missed something